Sources/_InternalTestSupport/SwiftTesting+TraitConditional.swift

```diff
@@ -29,6 +30,13 @@ extension Trait where Self == Testing.ConditionTrait {
         }
     }
 
+    /// Enabled only if toolchain support swift concurrency
```

Comment:
As a sidenote I think we're on new enough versions of macOS and Xcode everywhere in CI now to drop this entirely